netmap/netmap_contract.go

```diff
@@ -316,6 +317,8 @@ func UpdateState(state int, publicKey interop.PublicKey) {
 	default:
 		panic("unsupported state")
 	}
+
+	runtime.Notify("UpdateStateSuccess", state, publicKey)
```

Comment:
`UpdateStateIR` method should produce the same notification, I think.

Reply:
Done